test/e2e/utils/rbac.go

```diff
@@ -97,7 +98,15 @@ func ConfigureRBAC(ctx context.Context, cs clientset.Interface, ns string) error
 
 // DeconfigureRBAC removes RBAC configuration
 func DeconfigureRBAC(ctx context.Context, cs clientset.Interface, ns string) error {
-	err := cs.RbacV1().ClusterRoleBindings().Delete(ctx, "nfd-topology-updater-e2e", metav1.DeleteOptions{})
+	_, err := cs.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
```

Comment:
We still need to delete the cluster-scoped objects (ClusterRoles and ClusterRolebindings) even if the namespace is gone